src/main/java/org/springframework/metrics/instrument/DistributionSummary.java

```diff
@@ -63,10 +64,14 @@ default Builder tags(Iterable<Tag> tags) {
             return this;
         }
 
+        default Builder tags(String... tags) {
+            return tags(Tag.tags(tags));
```

Comment:
I thought about letting IntelliJ go hog wild with a static import. But then it would have been `tags(tags(tags))`.